npm/test-integration.js

```diff
@@ -61,8 +61,7 @@ module.exports = function (exit) {
                     collection: test.collectionJSON,
                     environment: test.environmentJSON,
                     globals: test.globalsJSON,
-                    iterationData: test.dataCSV || test.dataJSON,
-                    abortOnError: true
```

Comment:
If you don't send abort on error how will the Greg fail. It should have abortOnFailure